Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -70,7 +70,7 @@ def family
       end
 
       def flags
-        @flags ||= cpuinfo[/^(flags|Features).*/, 0]&.split
+        @flags ||= cpuinfo[/^(flags|Features).*/, 0]&.split&.drop(2)
```

Comment:
Could a comment be added to explain why these are being dropped? Similarly it might be nice to avoid the double `&` if possible.

Reply:
I can add a comment but I don't think removing the second `&` is an option --  Homebrew complains about an ordinary method following safe navigation operator without it:
```
Lint/SafeNavigationChain: Do not chain ordinary method call after safe navigation operator.
```

Reply:
```suggestion
        @flags ||= cpuinfo[/^(flags|Features)\s+: (.*)/, 2]&.split
```

Reply:
The format of the line is `flags` some number of tabs, a colon, a space, and then a space-separated list of words.

Reply:
Thanks, Shaun. Committed from my phone. Will add a comment when I get back home.